moveit_core/constraint_samplers/src/union_constraint_sampler.cpp

```diff
@@ -40,6 +40,8 @@
 
 namespace constraint_samplers
 {
+  rclcpp::Logger LOGGER_UNION_CONTRAINT_SAMPLERS = rclcpp::get_logger("moveit").get_child("constraint_samplers");
```

Comment:
no indent. Clang should catch this

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/abf4a74859bb5f3ca50a946b9bd0f48f1db59fa5